src/bundled.jsx

```diff
@@ -0,0 +1,38 @@
+/* eslint-disable react/jsx-props-no-spreading */
+import React from 'react';
+import ReactDOM from 'react-dom/client';
+import CioQuizComponent from './components/CioQuiz';
+import './styles.css';
+
+const CioQuiz = ({ selector, includeCSS = true, ...rest }) => {
+  if (document) {
+    const stylesheet = document.getElementById('cio-quizzes-styles');
+    const containerElement = document.querySelector(selector);
+
+    if (!containerElement) {
+      console.error(`ConstructorioQuizzes: There were no elements found for the provided selector`);
+
+      return;
+    }
+
+    if (stylesheet) {
+      if (!includeCSS) {
+        stylesheet.disabled = true;
+      } else {
+        stylesheet.disabled = false;
+      }
+    }
+
+    ReactDOM.createRoot(containerElement).render(
+      <React.StrictMode>
+        <CioQuizComponent {...rest} />
```

Comment:
Added `{...rest}` here so we don't forget to add new props to this when we make changes to the component

Reply:
Would there be any issues if the consumer passes in random "props"? 🤔